repo/fsrepo/fsrepo_test.go

```diff
@@ -30,31 +30,7 @@ func TestInitIdempotence(t *testing.T) {
 func TestRemove(t *testing.T) {
 	t.Parallel()
 	path := testRepoPath("foo", t)
-	assert.Nil(Remove(path), t, "should be able to remove after closed")
-}
-
-func TestCannotRemoveIfOpen(t *testing.T) {
```

Comment:
Yeah, you're right. the test's name was misleading. It didnt work across procs.

I think it's probably still a good idea to disable remove if another proc is holding the lock-- but that's not what this test did. We can do that in future PR.